internal/locate/cse/fallback.go

```diff
@@ -0,0 +1,153 @@
+package cse
+
+import (
+	"context"
+	"time"
+
+	"github.com/pingcap/kvproto/pkg/metapb"
+	"github.com/pingcap/log"
+	"github.com/sony/gobreaker"
+	pd "github.com/tikv/pd/client"
+	"go.uber.org/zap"
+)
+
+var (
+	_ pd.Client = &ClientWithFallback{}
+)
+
+type ClientWithFallback struct {
+	pd.Client
+	cse     pd.Client
```

Comment:
Why is it called `cse`？

Reply:
seems to be a typo, I will fix it.

Reply:
> Why is it called `cse`？

This field tends to store the CSEClient, which is an implementation of the interface PDClient.